labelbox/client.py

```diff
@@ -1136,12 +1187,12 @@ def _format_failed_rows(rows: List[str],
             dataRowsForGlobalKeys(where: {ids: $globalKeys}) { jobId}}
             """
         params = {"globalKeys": global_keys}
-        data_rows_for_global_keys_job = self.execute(query_str, params)
+        data_rows_for_global_keys_job = self.execute(query_str, params, timeout=180)
 
         # Query string for retrieving job status and result, if job is done
         result_query_str = """query getDataRowsForGlobalKeysResultPyApi($jobId: ID!) {
             dataRowsForGlobalKeysResult(jobId: {id: $jobId}) { data { 
-                fetchedDataRows { id }
+                fetchedDataRows { id createdAt updatedAt externalId globalKey rowData }
```

Comment:
based on the conversations yesterday, it sounds like this will revert back to just `id1